CMakeLists.txt

```diff
@@ -34,7 +31,20 @@ include(blt/SetupBLT.cmake)
 # Define RAJA PERFSUITE settings...
 #
 
+#cmake_dependent_option(RAJA_PERFSUITE_ENABLE_TESTS "Enable RAJA Perf Suite Tests" On "ENABLE_TESTS" Off)
+
+if (ENABLE_TESTS)
+
+  set(RAJA_PERFSUITE_ENABLE_TESTS On CACHE BOOL "Enable testing for RAJA Perf Suite")
+  set(RAJA_ENABLE_TESTS Off CACHE BOOL "")
+  set(CAMP_ENABLE_TESTS Off CACHE BOOL "")
+
+endif()
+
+set(ENABLE_EXAMPLES Off CACHE BOOL "Enable RAJA examples")
```

Comment:
Do we have a `RAJA_ENABLE_EXAMPLES` option that can be used instead?

Reply:
Actually, looks like that is set below and this line can be deleted.

Reply:
will do. thanks.